matlab/IceDiscovery/greeter/client.m

```diff
@@ -17,7 +17,7 @@ function client(args)
     % locator on the communicator.
     initData = Ice.InitializationData();
     initData.properties_ = Ice.createProperties(args);
-    initData.properties_.setProperty('Ice.Plugin.Discovery', 'IceDiscovery:createIceDiscovery');
```

Comment:
Note this worked when using "Discovery" as the name because we ship the plug-in dlls next to the toolbox dll. Unlike with Python where we have a single IcePy.so.